templates/post_buildspec.yml.tpl

```diff
@@ -14,7 +14,13 @@ phases:
       - yum install -y yum-utils
       - yum-config-manager --add-repo https://rpm.releases.hashicorp.com/AmazonLinux/hashicorp.repo
       - yum -y install consul
+      - yum -y install graphviz
+      - wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
```

Comment:
please switch to some official yum repo (as in line 15) - I prefer it from downloading arbitrary code from some github fork.  I think jq is well known and can be found easily.